processing/src/main/java/io/druid/segment/data/GenericIndexed.java

```diff
@@ -413,11 +414,11 @@ public int size()
 
     T bufferedIndexedGet(ByteBuffer copyValueBuffer, int startOffset, int endOffset)
     {
-      final int size = endOffset - startOffset;
+      final int size = endOffset > startOffset
+                       ? endOffset - startOffset
+                       : copyValueBuffer.get(startOffset - Ints.BYTES);
```

Comment:
Integer.BYTES

Reply:
How is this going to make any difference ?

Reply:
Consistency

Reply:
I am not sure which consistency are we talking about here. I think there is nothing wrong with using Integer.BYTES or Ints.BYTES, when both are essentially same and both are already used in druid at many places. 
If you look at the complete GenericIndexed class Ints.BYTES is used at lots of places, So i think it should be ok to use the same here. 

It would be great if we please refrain from adding comments that do not contribute in a meaningful way in improving the PR, but they are just there to line up with whatever personal coding preference one has. 

Another way to add such comments is to prefix them with "optional:..." so that the PR owner knows that he can choose to handle or ignore the comment without blocking his PR.